sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Clauses/WhereClause.cs

```diff
@@ -8,20 +8,28 @@ namespace Azure.DigitalTwins.Core.QueryBuilder
     /// <summary>
     /// Custom object for a WHERE clause. Only meant to be used when adding WHERE to a query. Hidden from user.
     /// </summary>
-    internal class WhereClause : ClauseBase
+    internal class WhereClause
     {
         /// <summary>
-        /// Condition object that encodes the logical condition behind the WHERE clause.
+        /// Condition object represented in string format that encodes the logical condition behind the WHERE clause.
         /// </summary>
-        internal ConditionBase Condition { get; set; }
+        internal string Condition { get; set; }
 
         /// <summary>
-        /// Constructor for a WHERE clause.
+        /// Constructor for a WHERE clause using the internal condition object.
         /// </summary>
         /// <param name="condition"> Condition argument for the WHERE clause. </param>
         internal WhereClause(ConditionBase condition)
         {
-            Type = ClauseType.WHERE;
+            Condition = condition.Stringify();
+        }
+
+        /// <summary>
+        /// Constructor for a WHERE clause allowing for user formed strings and prebuilt ADT WHERE function arguments.
```

Comment:
`pre-built ADT conditional statements`

Reply:
How about `Constructor for a WHERE clause that allows for overridden conditional statements in string format`?